testsuite/RealNDTest/CMakeLists.txt

```diff
@@ -1,5 +0,0 @@
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../../src/include)
-add_definitions (-DBOOST_TEST_DYN_LINK)
-add_executable(RealNDTest RealNDTest.cpp)
```

Comment:
Not sure we can remove this file as it is non-standard.

Reply:
The include should be irrelevant as nothing is included from `src/include`. Therefore, it should be "standard".

Reply:
Ok, got it, the blob should work for that test as well